packages/vscode-extension/src/service/TranslationService.ts

```diff
@@ -0,0 +1,31 @@
+/*
+** Copyright (c) 2019 Oracle and/or its affiliates.  All rights reserved.
+** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+*/
+const ApplicationConstants = require('../../../node-cli/src/ApplicationConstants');
+const FileUtils = require('../../../node-cli/src/utils/FileUtils');
+const path = require('path');
+let MESSAGES: { [x: string]: any; };
+
+export class TranslationService {
```

Comment:
Isn't it possible to use TranslationSerice from node-cli package?

Reply:
The thing is that I want to separate the messages.json file from Node-cli and vscode extension

I can modify TranslationService from node-cli package to point to different files. What do you think?

Reply:
Done